src/analysis/stack-lattice.h

```diff
@@ -0,0 +1,171 @@
+#ifndef wasm_analysis_stack_lattice_h
+#define wasm_analysis_stack_lattice_h
+
+#include <deque>
+
+#include "lattice.h"
+
+namespace wasm::analysis {
+
+// A lattice for modelling stack-related phenomena. It is templated
+// on a lattice representing the contents of each stack frame.
```

Comment:
Perhaps some examples would be useful here. Starting to read this file, I'm not sure if stack here means the call stack, or the wasm value stack, or a generic stack of anything unrelated to wasm, or otherwise.

Reply:
An example with used bits was added.